### PR TITLE
KOGITO-9257: Add a breadcrumb in the settings pages to go back to home page

### PR DESCRIPTION
**Pull request by fantonangeli**
Jira: https://issues.redhat.com/browse/KOGITO-9257

**Description**
Currently, the only way how to get from settings is to click the main logo.
It might be confusing for the users to get back to the main navigation.

[KOGITO-9257.webm](https://github.com/kiegroup/kie-tools/assets/17780574/c596db60-9b36-4e2d-9019-ad467d95e634)

### CODE REVIEW

File: packages/serverless-logic-web-tools/src/settings/github/GitHubSettings.tsx [hunk 1]
```diff
@@ -100,24 +99,19 @@ export function GitHubSettings(props: SettingsPageProps) {
...
-            All information is locally stored in your browser and never shared with anyone.
-          </Text>
-        </TextContent>
-      </PageSection>
-
+    <SettingsPageContainer
+      pageTitle={PAGE_TITLE}
+      subtitle={
+        <>
+          {" "}
```
Comment (caponetto) [inline packages/serverless-logic-web-tools/src/settings/github/GitHubSettings.tsx]: Is `{" "}` necessary?

File: packages/serverless-logic-web-tools/src/settings/serviceAccount/ServiceAccountSettings.tsx [hunk 2]
```diff
@@ -99,22 +98,16 @@ export function ServiceAccountSettings(props: SettingsPageProps) {
...
-          </Text>
-        </TextContent>
-      </PageSection>
-
+    <SettingsPageContainer
+      pageTitle={PAGE_TITLE}
+      subtitle={
+        <>
+          Data you provide here is necessary for uploading Open API specs associated with models you design to your
+          Service Registry instance. All information is locally stored in your browser and never shared with anyone.
```
Comment (caponetto) [inline packages/serverless-logic-web-tools/src/settings/serviceAccount/ServiceAccountSettings.tsx]: Can you please add a `<br />` before "All information...." like we have in GitHub and OpenShift settings?

File: packages/serverless-logic-web-tools/src/settings/serviceRegistry/ServiceRegistrySettings.tsx [hunk 3]
```diff
@@ -98,22 +97,16 @@ export function ServiceRegistrySettings(props: SettingsPageProps) {
...
-          </Text>
-        </TextContent>
-      </PageSection>
-
+    <SettingsPageContainer
+      pageTitle={PAGE_TITLE}
+      subtitle={
+        <>
+          Data you provide here is necessary for uploading specs associated with models you design to your Service
+          Registry instance. All information is locally stored in your browser and never shared with anyone.
```
Comment (caponetto) [inline packages/serverless-logic-web-tools/src/settings/serviceRegistry/ServiceRegistrySettings.tsx]: Can you please add a `<br />` before "All information...." like we have in GitHub and OpenShift settings?

**Review by caponetto — APPROVED** [review]

**Review by tomasdavidorg — APPROVED** [review]
Looks good to me. Thanks 🙂.

**Pull request merged**